tests/plotting/test_plotting.py

```diff
@@ -264,6 +264,10 @@ def test_plot_update(sphere):
     pl.close()
 
 
+# https://github.com/pyvista/pyvista/pull/4950
+@pytest.mark.skipif(
+    platform.system() == 'Linux', reason='Does not display correctly within latest ubuntu'
+)
```

Comment:
We should avoid these sorts of skips and instead skip via the `verify_image_cache` fixture